dash/dash.py

```diff
@@ -2076,3 +2170,137 @@ def verify_url_part(served_part, url_part, part_name):
+            warnings.warn(
+                f"A folder called {self.pages_folder} does not exist.", stacklevel=2
+            )
+        if os.path.exists(self.config.pages_folder):
+            self._import_layouts_from_pages()
+
+        @self.server.before_first_request
+        def router():
+            @self.callback(
+                Output(_ID_CONTENT, "children"),
+                Output(_ID_STORE, "data"),
+                Input(_ID_LOCATION, "pathname"),
+                Input(_ID_LOCATION, "search"),
+                prevent_initial_call=True,
+            )
+            def update(pathname, search):
+                """
+                Updates dash.page_container layout on page navigation.
+                Updates the stored page title which will trigger the clientside callback to update the app title
+                """
+
+                # update page registry for pages that might have been added or changed in a callback
+                self.page_registry = _pages.PAGE_REGISTRY.copy()
+
+                query_parameters = _parse_query_string(search)
+                page, path_variables = self._path_to_page(
+                    self.strip_relative_path(pathname)
+                )
+
+                # get layout
+                if page == {}:
+                    if "pages.not_found_404" in self.page_registry:
+                        layout = self.page_registry["pages.not_found_404"]["layout"]
+                        title = self.page_registry["pages.not_found_404"]["title"]
+                    else:
+                        layout = html.H1("404")
+                        title = self.title
+                else:
+                    layout = page["layout"]
+                    title = page["title"]
+
+                if isinstance(layout, patch_collections_abc("Callable")):
+                    layout = (
+                        layout(**path_variables, **query_parameters)
+                        if path_variables
+                        else layout(**query_parameters)
+                    )
+                if isinstance(title, patch_collections_abc("Callable")):
+                    title = title(**path_variables) if path_variables else title()
+
+                return layout, {"title": title}
+
+            _validate.check_for_duplicate_pathnames(self)
+
+            # Set validation_layout
+            self.validation_layout = html.Div(
+                [
+                    page["layout"]() if callable(page["layout"]) else page["layout"]
+                    for page in self.page_registry.values()
+                ]
+                + [self.layout]
+            )
+            if _ID_CONTENT not in self.validation_layout:
+                raise Exception("`dash.page_container` not found in the layout")
+
+            # Update the page title on page navigation
+            self.clientside_callback(
+                """
+                function(data) {{
+                    document.title = data.title || 'Dash'
```

Comment:
I just removed the `|| 'Dash'` because data.title is updated [here](https://github.com/plotly/dash/blob/2780bb6d1a66535a8f3799d0c800ab7852d3efd6/dash/dash.py#L2269)